### PR TITLE
Replace `renderCachedPlot()` code with `bindCache()`

### PR DESCRIPTION
**Pull request by wch**
This replaces the underlying implementation for `renderCachedPlot()`, with `bindCache(renderPlot())`.

### CODE REVIEW

File: R/render-cached-plot.R [hunk 1]
```diff
@@ -216,256 +217,19 @@ renderCachedPlot <- function(expr,
...
-          img$height <- NULL
-
-          img
-        })
-      }
+  inject(
+    bindCache(
+      renderPlot(!!expr, res = res, alt = alt, outputArgs = outputArgs),
+      !!cacheKeyExpr,
+      sizePolicy = sizePolicy
```
Comment (schloerke) [inline R/render-cached-plot.R:230]: `cache` is not used anymore

Comment (schloerke) [reply R/render-cached-plot.R:230]: Neither is `width` or `height`

Comment (schloerke) [reply R/render-cached-plot.R:230]: `...` are also ignored

**Review by schloerke — CHANGES_REQUESTED** [review]
Either deprecate or document new changes for `...`, `width`, `height`, and `cache`.

File: R/render-cached-plot.R (hunk 1 above)
Comment (wch) [reply R/render-cached-plot.R:230]: Thanks, it now passes along `cache` and `...`. The `width` and `height` are meant to be ignored.

**Comment by wch** [comment]
Regarding docs: there's (now) no change in the behavior of the args, so I don't think the documentation needs to be changed.

File: R/render-cached-plot.R [hunk 2]
```diff
@@ -216,256 +217,20 @@ renderCachedPlot <- function(expr,
...
-  # This is so that the expr doesn't re-execute by itself; it needs to be
-  # triggered by the cache key (or width/height) changing.
-  isolatedFunc <- function() isolate(func())
-
-  args <- list(...)
-
-  if (!is.null(width) || !is.null(height)) {
-    warning("Unused argument(s) 'width' and/or 'height'. ",
-            "'sizePolicy' is used instead.")
-  }
```
Comment (schloerke) [inline R/render-cached-plot.R:231]: Restore this code, please

**Review by schloerke — APPROVED** [review]
Given height and width value check code being restored, LGTM

File: R/render-cached-plot.R (hunk 2 above)
Comment (wch) [reply R/render-cached-plot.R:231]: Done

**Pull request merged**